### DISCUSSION TITLE
Athom: Potential GPL violation

### DISCUSSION THREAD

**Discussion by voltagex** [arendst/Tasmota#19453] [General]
Athom are running a customised Tasmota on their gear, but when I asked about sources I got pointed to here.

**Comment by s-hadinger** [comment]
Is it really a customized version, or did they only change the footer?

**Reply by voltagex** [reply]
> Is it really a customized version, or did they only change the footer?

Isn't changing the footer enough?

We're also missing information as to what configuration and version was used - important for things like the Zigbee bridge

**Reply by s-hadinger** [reply]
> Isn't changing the footer enough?

In theory yes. But that would not bring very interesting information.

**Reply by Jason2866** [reply]
Real world view. Athom is a company in China. Even big companys with many Lawers have no way to force companys in China to stop what the are doing.
**Tasmota is in no way related to Athom**

**Reply by sfromis** [reply]
Any reason to assume other changes to Tasmota than the footer?

When it comes to information about configuration and version, you can get a long way by checking what's available on the "Information" page, via `Status` commands, and especially the tools decode-status and decode-config.

**Reply by barbudor** [reply]
> We're also missing information as to what configuration and version was used - important for things like the Zigbee bridge

As far as I know, Athom publishes Tasmota configuration details for their product on most products pages. Just picking up randomly a socket on their web site: https://fr.athom.tech/blank-1/tasmota-uk-wall-socket
For the Zigbee bridge, it is documented in the Tasmota Templates repository : https://templates.blakadder.com/athom_ZG01.html

They clearly state that their devices is fully compatible with Tasmota OTA server which means it can run stock Tasmota firmware without a needing any code change.

Changing the name of the company in the footer is not really a big deal as it goes away at the 1st upgrade, and as far as I remember, they have always been quite cooperative and even joined some discussions here.

I remember some others companies that made heavy changes in Tasmota and were not so much cooperative.

**Reply by voltagex** [reply]
Fair enough. Thanks
